docs/MAINTENANCE.md

```diff
@@ -5,3 +5,17 @@ Follow this guide to learn about the maintainance tasks of the repository.
 * [Dependabot Updates](#dependabot-updates)
 
 ## Dependabot Updates
+
+PRs sent by Dependabot can be found here - https://github.com/wso2/identity-apps/pulls/app%2Fdependabot
+
+Before merging Dependabot PRs, they need to be tested and verified that they don't introduce any bugs or breaking changes locally.
+
+1. Fetch the latest codebase from upstream.
+
+2. **Locally** merge the dependabot branch.
+
+3. Test the changes and verify that everything is working in the product.
+
+4. If the changes are non-breaking, add a comment to verify the testing of the Dependabot PR and merge the PR.
```

Comment:
Since `dependabot` bumps the `package.lock.json` versions, if the dependency in question is a direct dependency, we can bump the version in `package.json`  and send another PR closing the dependabot PR.

If it's a transitive dependency, we can check whether we can bump the parent depency. In case if the parent dependency doesn't have the fixed version of the transitive dependency, we can create an issue in the respective project and merge the `package.lock.json` as a temp fix.

Reply:
Fixed with 05e6a87f972b0bdd3c6edd3991cf696b4cbf774a